Package.swift

```diff
@@ -14,42 +14,42 @@ let package = Package(
     targets: [
         .binaryTarget(
             name: "ffmpegkit",
-            url: "https://github.com/Truvideo/truvideo-sdk-ios-video-utils/releases/download//ffmpegkit.xcframework.zip",
```

Comment:
@federico-amura-kenility I think something might have broken on bitrise. The version was removed in the last [release 73.1.1](https://github.com/Truvideo/truvideo-sdk-ios-video/commit/1b64e1ca07a026012c047c1bccc7418e4f47eb7b)